### ISSUE TITLE
Cleanup references to PRDD

### ISSUE THREAD

**Issue by holdenk** [sparklingpandas/sparklingpandas#70]
We call our DataFrames' PRDDs' in a number of cases (mostly in terms of as inner variables). We should clean this up now that we renamed PRDDs' to DataFrames'.

**Comment by michalmonselise** [comment]
Is it ok to assume that this doesn't apply to prdd.py and the PRDD class?

**Comment by holdenk** [comment]
So this applies against the v0.0.2 branch (I'll update the milestone)

**Comment by holdenk** [comment]
Once we get v0.0.2 merged in to master prdd.py will be replaced with dataframe.py but there are a bunch of old references to prdd hanging around (mostly just in test names & pydocs) that we should cleanup. I can make sure to ping this ticket once v0.0.2 is merged into master if you ( @michalmonselise ) are interested in helping with it :)

**Comment by holdenk** [comment]
@michalmonselise We've got v0.0.2 in master now so if you want to tackle this (or some of this) that would be rad :)

**Comment by michalmonselise** [comment]
Cool.
I'll work on it as soon as possible.

**Comment by holdenk** [comment]
Rad, no rush :)
